src/engine/sync/enginesync.cpp

```diff
@@ -356,3 +358,34 @@ void EngineSync::deactivateSync(Syncable* pSyncable) {
         }
     }
 }
+
+EngineChannel* EngineSync::pickLegacySyncTarget(EngineChannel* pDontPick) const {
```

Comment:
I don't like the name legacy, since that implies it will get removed and I don't foresee a day when there will *only* be sync-enabled decks.  pickNonsyncSyncTarget?